src/entry-helpers.ts

```diff
@@ -39,7 +49,12 @@ export const getConfigFromPage = (page: string) => {
   return Object.fromEntries(entries);
 };
 
-export const pushBullets = async (bullets: string[]) => {
+export const pushBullets = async (bullets: string[], title: string = "") => {
+  // There is an issue with periods which happens only for the first bullet. 
+  // Deliberrately adding a period, so that subsequent bullets don't face that issue.
+  bullets.unshift(title + ".");
+
+  // TODO: Nest bullets under title bullet.
```

Comment:
why's this?

Reply:
I thought it'd be good to nest whatever we add to roam under the title bullet for better organization.
But, I just realised the user can create a nested bullet themselves before using the extensions, so not really a necessity.